projects/sonar/src/app/_layout/admin/admin.component.ts

```diff
@@ -53,6 +53,21 @@ export class AdminComponent implements OnInit, OnDestroy {
     private _translateService: TranslateService
   ) {}
 
+  /**
+   * Administration menu entries
+   */
+  get administrationMenu(): string[] {
+    let sections = ['collections', 'organisations', 'subdivisions', 'users'];
+    /** Delete menu entries if the organization is shared */
+    if (this._userService.isSharedOrganisation()) {
+      sections = sections.filter(section => !(['collections', 'subdivisions'].includes(section)));
```

Comment:
Why not return a static list instead of using a filter?

Reply:
To keep only one entries list in the menu (first line) and remove entries after.